core/lib/search/adapter/api/searchAdapter.ts

```diff
@@ -75,14 +76,36 @@ export class SearchAdapter {
     url = url + '/' + encodeURIComponent(Request.index) + '/' + encodeURIComponent(Request.type) + '/_search'
     url = url + '?' + queryString.stringify(httpQuery)
 
+    let queryMethod = config.elasticsearch.queryMethod
+    if (Request.type === 'product' && Request.sort) {
+      const sortType = Request.sort.split(':', 2)
+      if (sortType[0] === 'position') {
+        const currentCategoryId = store.getters['category-next/getCurrentCategory']['id']
+        if (currentCategoryId) {
+          // It's mandatory to send a POST request, in this case, otherwise, this sorting won't work
```

Comment:
Why? We won't benefit from browser's caching

Reply:
Because otherwise, with a GET request, it doesn't work.